src/main/java/org/thoughtslive/jenkins/plugins/jira/steps/AssignIssueStep.java

```diff
@@ -87,14 +87,13 @@ protected Execution(final AssignIssueStep step, final StepContext context)
 
```

Comment:
Change to this class alone is sufficient to un-assign issue. I don't think we need to introduce the custom class - AssigneeParameter. Please remove and try to test it.

Reply:
Unfortunately the ImmutableMap can't handle null values. Do you have another idea?

Reply:
Good catch!, I forgot about that part. lets try the normal map then. 

```java
  public ResponseData<Void> assignIssue(final String issueIdorKey, final String userName) {
    try {
      Map input = Maps.newHashMap();
      input.put("name", userName);
      return parseResponse(
          jiraEndPoints
              .assignIssue(issueIdorKey, input)
              .execute());
    } catch (Exception e) {
      return buildErrorResponse(e);
    }
  }
```

Reply:
Good idea! It works the way it should.